README.md

```diff
@@ -21,8 +21,8 @@ Quick Start
 
 To build the JARs, use the commands:
 
-    cd lucene
-    ant -Dmobilejars=../build/<destination> clean mobile-build-modules-without-test
+    $> cd lucene
```

Comment:
In all my README I add the `$>` to emphase when a highlighted block is a command line sample or a code one. The distinction is not clear enough on Github (imo) and in case the documentation tends to be complicated (with scripts and code samples) I rather add some (maybe) unnecessary markers than confuse a potential reader.
Nonetheless, I have update the current README following your suggestion. :bowtie: